modules/tensor_mechanics/src/actions/PressureAction.C

```diff
@@ -42,11 +42,18 @@ validParams<PressureAction>()
   params.addParam<Real>("factor", 1.0, "The factor to use in computing the pressure");
   params.addParam<Real>("alpha", 0.0, "alpha parameter for HHT time integration");
   params.addParam<FunctionName>("function", "The function that describes the pressure");
+  params.addParam<AuxVariableName>("pressure", "The auxiliary variable that contains the pressure");
   return params;
 }
 
 PressureAction::PressureAction(const InputParameters & params) : Action(params)
 {
+  if (isParamValid("pressure") && isParamValid("function"))
+    mooseError("BCs/Pressure/",
```

Comment:
Rather than hardcode this, perhaps use paramError() instead? You'd only be able to point to one of the two troublesome params but that would probably be cleaner and give more context to the error message (e.g. the line number).

Reply:
> You'd only be able to point to one of the two troublesome params

Yeah, that's why went with `mooseError`. I can do `paramError()` instead...